samples/word/Basics/basic-doc-assembly.yaml

```diff
@@ -0,0 +1,229 @@
+id: ciyvyru7l00014r6ydkzkwka7
+gist: ''
+source: WORD
+author: Juan Balmori - Microsoft
+name: Basic document assembly.
+description: Basic document assembly capabilities of the Word JavaScript API. Learn the basic objects and usage.
+script:
+  content: |
+    $("#insert-header").click(insertHeader);
+    $("#add-paragraphs").click(addParagraphs);
+    $("#add-content-controls").click(addContentControls);
+    $("#change-customer").click(changeCustomer);
+    $("#add-footer").click(addFooter);
+
+  // Inserts a heading 1 Title.
+    function insertHeader() {
+        Word.run(function (context) {
+            var header = context.document.body.insertText("This is a sample Heading 1 Title!!", "start");
+            header.styleBuiltIn = Word.Style.heading1;
+            return context.sync();
+        })
+        .catch(function (error) {
+            console.log(error.message);
+        })
+    }
+
+     // Insert a set of paragraphs, a content control and a page break.
+        function addParagraphs() {
+            Word.run(function (context) {
+                // Queue a command to add a paragraph at the end of the document. Set the style on the paragraph.
+                var paragraph = context.document.body.insertParagraph("Timeline", "end");
+                paragraph.style = "Heading 2";
+                var paragraph2 = context.document.body.insertParagraph("The Services shall commence on July 31, 2015, and shall continue through July 29, 2015.", "end");
+                paragraph2.style = "Normal";
+
+                var paragraph3 = context.document.body.insertParagraph("Project Costs by Phase", "end");
+                paragraph3.style = "Heading 2";
+
+                // Queue a command to add a paragraph at the end of the document. Set the style on the paragraph and wrap the paragraph
+                // in a content control with the title of "ProjectCosts".
+                var paragraph4 = context.document.body.insertParagraph("<Add Project Costs Here>", "end");
+    paragraph4.style = "Normal";

```

Comment:
@WrathOfZombies, I'm pretty sure Juan would have indented correctly when he wrote the snippet, so do you now why the indentation is lost here?